djongo/schema.py

```diff
@@ -3,8 +3,17 @@
 
 logger = getLogger(__name__)
 
+
 class DatabaseSchemaEditor(BaseDatabaseSchemaEditor):
+
     sql_create_index = "ALTER TABLE %(table)s ADD CONSTRAINT %(name)s INDEX (%(columns)s)%(extra)s"
+    sql_delete_index = "DROP INDEX %(name)s ON %(table)s"
+
+    def quote_value(self, value):
+        raise NotImplementedError()
+
+    def prepare_default(self, value):
+        raise NotImplementedError()
```

Comment:
These are just methods that were not implemented in the parent class. Adding `NotImplementedError` here to make it a bit more explicit.